src/Writers/HTMLWriter.jl

```diff
@@ -796,6 +796,13 @@ function render_html(ctx, navnode, head, sidebar, navbar, article, footer, scrip
     )
 end
 
+function documenter_version()
+    project = joinpath(dirname(dirname(pathof(Documenter))), "Project.toml")
```

Comment:
I think it would be better to do this at compile time since at runtime we might not have access to this file (eg if we are running from a sysimage)

Reply:
https://github.com/JuliaDocs/Documenter.jl/pull/1611